src/components/AddButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Text} from 'react-native';
+import styled from 'styled-components/native';
+
+const Container = styled.View`
```

Comment:
styles should be stored in a separate file, the most common way to do this is with this format:

```
src/
    Components/
      Container/
        index.tsx <---- this file
        styles.ts <---- move the styles here
```